core/embed/rust/src/ui/model_tr/component/input_methods/choice.rs

```diff
@@ -156,9 +171,33 @@ where
     }
 
     /// Navigating to the chosen page index.
-    pub fn set_page_counter(&mut self, ctx: &mut EventCtx, page_counter: usize) {
-        self.page_counter = page_counter;
-        self.update(ctx);
+    pub fn set_page_counter(
+        &mut self,
+        ctx: &mut EventCtx,
+        page_counter: usize,
+        do_animation: bool,
```

Comment:
It is currently used only twice and is `true` in both cases.

I would still keep the boolean argument, so that it can support non-animation any-time and acts as a self-documentation that it is even doing the animation (without it, it might not be quickly apparent).

Reply:
My reason for asking is that without the argument it's obvious what the method does (even at the place where it's called), though with it I had to look it up. But whatever, maybe it's just my editor that sucks.